uax/uax42/Readme.md

```diff
@@ -0,0 +1,24 @@
+# Generating TR42
+
+## Step 1 - Generate property value fragments
+
+- Run org.unicode.xml.GeneratePropertyValues to populate the UNICODETOOLS_REPO_DIR/uax/uax42/fragments/ folder.
+
+## Step 2 - Generate TR42 index.html and index.rnc 
+
+- In UNICODETOOLS_REPO_DIR/uax/uax42/ run `mvn xml:transform`
+
+  index.html and index.rnc will be generated in UNICODETOOLS_REPO_DIR/uax/uax42/output/
+
+## Step 3 - Validate generated UAX XML files
+
+You'll need a [RELAX NG](https://relaxng.org/) schema validator. We'll use [jing-trang](https://github.
+com/relaxng/jing-trang) in this example.
+
+1. Clone and build [jing-trang](https://github.com/relaxng/jing-trang)
+2. Run the following:
+    ```
+   java -jar C:\_git\jing-trang\build\jing.jar -c UNICODETOOLS_REPO_DIR\uax\uax42\output\index.rnc <path to UAX xml file>
```

Comment:
mvn ...

Reply:
I think that we should probably discuss what is in-scope and what is out-of-scope for the process and the utility. The deliverables are:
1. The UCDXML files
2. The TR42 HTML file
3. The RNC file

The fragment files are generated, but are then consumed by the TR42 HTML file and the RNC file. I *think* that these should be in the repo, especially as some of them are created manually.

Should any of the deliverables be stored in unicodetools?
If we are planning to validate the UCDXML files using the RNC files as part of the build, we'll need a process that incorporates all steps. Should that be considered a "unit test"?